packages/block-library/src/embed/embed-controls.js

```diff
@@ -28,37 +34,57 @@ const EmbedControls = ( {
 	allowResponsive,
 	toggleResponsive,
 	switchBackToURLInput,
-} ) => (
-	<>
-		<BlockControls>
-			<ToolbarGroup>
-				{ showEditButton && (
-					<ToolbarButton
-						className="components-toolbar__control"
-						label={ __( 'Edit URL' ) }
-						icon={ edit }
-						onClick={ switchBackToURLInput }
-					/>
-				) }
-			</ToolbarGroup>
-		</BlockControls>
-		{ themeSupportsResponsive && blockSupportsResponsive && (
-			<InspectorControls>
-				<PanelBody
-					title={ __( 'Media settings' ) }
-					className="blocks-responsive"
-				>
-					<ToggleControl
-						__nextHasNoMarginBottom
-						label={ __( 'Resize for smaller devices' ) }
-						checked={ allowResponsive }
-						help={ getResponsiveHelp }
-						onChange={ toggleResponsive }
-					/>
-				</PanelBody>
-			</InspectorControls>
-		) }
-	</>
-);
+} ) => {
+	const dropdownMenuProps = useToolsPanelDropdownMenuProps();
+
+	return (
+		<>
+			<BlockControls>
+				<ToolbarGroup>
+					{ showEditButton && (
+						<ToolbarButton
+							className="components-toolbar__control"
+							label={ __( 'Edit URL' ) }
+							icon={ edit }
+							onClick={ switchBackToURLInput }
+						/>
+					) }
+				</ToolbarGroup>
+			</BlockControls>
+			{ themeSupportsResponsive && blockSupportsResponsive && (
+				<InspectorControls>
+					<ToolsPanel
+						label={ __( 'Media settings' ) }
+						resetAll={ () => {
+							toggleResponsive( ! allowResponsive );
+						} }
+						dropdownMenuProps={ dropdownMenuProps }
+					>
+						<ToolsPanelItem
+							label={ __( 'Media settings' ) }
+							isShownByDefault
+							hasValue={ () =>
+								! allowResponsive || ! blockSupportsResponsive
+							}
+							onDeselect={ () => {
+								toggleResponsive( ! allowResponsive );
+							} }
+						>
+							<ToggleControl
+								__nextHasNoMarginBottom
+								label={ __( 'Resize for smaller devices' ) }
+								checked={ allowResponsive }
+								help={ getResponsiveHelp }
+								onChange={ ( v ) => {
+									toggleResponsive( v );
+								} }
```

Comment:
```suggestion
								onChange={ toggleResponsive }
```

Nit: Don't need to use the arrow function